nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexMojo.java

```diff
@@ -108,6 +103,20 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
     }
 
+    private void executeHelper(int seedNum) {
+        NonDexSurefireExecution execution =
```

Comment:
Here you would need to make some changes, so that you pass nondexStart and nondexEnd.